finviz/helper_functions/scraper_functions.py

```diff
@@ -23,7 +23,7 @@ def get_table(page_html: requests.Response, headers, rows=None, **kwargs):
     # Skip the first element ([1:]), since it's the headers (we already have it as a constant)
     all_rows = [
         column.xpath("td//text()")
-        for column in page_parsed.cssselect('tr[valign="top"]')[1 : rows + 1]
+        for column in page_parsed.cssselect('tr[valign="top"]')[0 : rows + 1]
```

Comment:
I'd just do
```suggestion
        for column in page_parsed.cssselect('tr[valign="top"]')
```
Also remove the comment since it's no longer true?